keras_model.py

```diff
@@ -0,0 +1,36 @@
+from keras.models import load_model
+from multiprocessing import Lock
+from multiprocessing.managers import BaseManager
+
+import config
+cfg = config.get_localized_config()
+
+class KerasModel():
+    def __init__(self):
+        self.mutex = Lock()
+        self.model = None
+
+    def initialize(self):
+        self.model = load_model(cfg.model_path)
+        self.model._make_predict_function()
+
+    def predict(self, input):
+        with self.mutex:
+            return self.model.predict(input)
+
+class KerasManager(BaseManager):
+    pass
+
+KerasManager.register('KerasModel', KerasModel)
+
+if __name__ == '__main__':
```

Comment:
Optional, but maybe remove this line and the ones below, unless you want them for debugging. They should only run if you invoke this file from the command line, which I guess you don't do.